docs/source/content/create.drip.rst

```diff
@@ -16,11 +16,20 @@ On the other hand, when you click in the ``FIELD NAME OF USER`` input, you will
 
 Here you can relate the Drip to the corresponding ``Campaign``. Grouping several drips under a campaign.
 
+Lookup fields
+-------------
+The fields you can make a query are User model fields and fields of models related to the user model. 
```

Comment:
```suggestion
You can make queries using the User's model fields and the fields of the models related to the user.
```

Reply:
Added this suggestion because "The fields you can make a query" is missing something

Reply:
Sounds and looks better, thanks!